src/__tests__/organizations.int.test.ts

```diff
@@ -0,0 +1,217 @@
+import request from 'supertest';
+import { app } from '../app';
+import { createOrganization, db, loadTableMetrics } from '../database';
+import { expectTimestamp } from '../test/utils';
+import { mockJwt as authHeader } from '../test/mockJwt';
+import { PostgresErrorCode } from '../types';
+
+const agent = request.agent(app);
+
+const insertTestOrganizations = async () => {
+	await createOrganization({
+		employerIdentificationNumber: '11-1111111',
+		name: 'Example Inc.',
+	});
+	await createOrganization({
+		employerIdentificationNumber: '22-222222',
```

Comment:
While we don't have format validation, it appears this is meant to mimic a real EIN, which means it should be nine digits. (I'm pretty confident you intended this, and e.g. `11-1111111` conforms properly.)

Only flagging it once but I see it several times:

```suggestion
		employerIdentificationNumber: '22-2222222',
```